src/main/java/hello/helper/GetResponse.java

```diff
@@ -0,0 +1,33 @@
+package hello.helper;
```

Comment:
Instead of creating so many response classes, we can use `ResponseEntity<Map>` and populate the map with appropriate data.